src/strategies/snet-liquidity-providers/index.ts

```diff
@@ -0,0 +1,76 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, call } from '../../utils';
+
+export const author = 'bonustrack';
```

Comment:
```suggestion
export const author = 'Vivek205';
```

Reply:
fixed in https://github.com/snapshot-labs/snapshot-strategies/pull/202/commits/82d0311e9678f0aa1db49d9b29a5d7f7e71c5403